File: responses/YXVDK8-JB4/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(X):
    X = str(X)

    if '-' in X:
        X = X.replace('-', '')
        X = X[::-1]
        X = '-' + X
    else:
        X = X[::-1]
        X = X

    return int(X)

File: responses/Y3VKJF-D4A/reverse_digits.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(x)
l = m = 0

while x > 0 do 

m = m + x if x%5 == 0 && !x.eql?(5)
l = l + x if x%3 == 0 && !x.eql?(3)
x -=1
end


l+m
end